test/core/security/grpc_tls_certificate_provider_test.cc

```diff
@@ -495,8 +495,54 @@ TEST_F(GrpcTlsCertificateProviderTest,
   CancelWatch(watcher_state_1);
 }
 
-}  // namespace testing
+TEST_F(GrpcTlsCertificateProviderTest, EmptyPrivateKeyString) {
+  std::string empty_string;
+  absl::Status match_status =
+      PrivateKeyPublicKeyMatches(empty_string, cert_chain_);
```

Comment:
Here you can avoid empty_string variable.
The function call could be
absl::Status status = PrivateKeyPublicKeyMatches(/*private_key_string=*/"", cert_chain_);
Similarly for cert_chain below

Reply:
Noted, thanks!